src/application/controllers/signup.ts

```diff
@@ -33,7 +33,7 @@ export class SignUpController implements Controller {
         email,
         phone,
         password,
-        verificationToken: 'token dumb'
+        verificationToken: ''
```

Comment:
so are we going to store a user with an empty verification token? When will this change?

Reply:
when confirmation email logic is implemented. Because users will need click on email link to confirm with this value.

Reply:
I understand, thanks for the answer!!